flow-typed/forms.js

```diff
@@ -130,14 +130,23 @@ declare type CheckboxElement = FormElementWithOptionsBase & {
   buttons: boolean,
   readOnly: boolean,
   defaultValue?: ?(string[]),
+  canSelectAll?: boolean,
 }
 
+type SelectMulti =
+  | {
+      multi: true,
+      canSelectAll?: boolean,
```

Comment:
is `canToggleAll` more appropriate? Im happy to merge if you are :)

Reply:
Yeh, I'll change it